packages/expo-firebase-analytics/ios/EXFirebaseAnalytics/EXFirebaseAnalytics.m

```diff
@@ -14,15 +14,15 @@ @implementation EXFirebaseAnalytics
                     resolver:(EXPromiseResolveBlock)resolve
                     rejecter:(EXPromiseRejectBlock)reject) {
     [FIRAnalytics logEventWithName:name parameters:props];
-    resolve(nil);
+    resolve([NSNull null]);
```

Comment:
curious about these changes -- why explicitly return `null` instead of `undefined`?

Reply:
https://github.com/expo/universe/pull/3145#discussion_r217884159